packages/pg/lib/client.js

```diff
@@ -577,9 +579,6 @@ class Client extends EventEmitter {
                 Client.failedHostsTime.set(this.host, start)
                 Client.connectionMap.delete(this.host)
                 Client.hostServerInfo.delete(this.host)
-              } else if (Client.failedHosts.has(this.host)) {
-                Client.failedHosts.delete(this.host)
-                Client.failedHostsTime.delete(this.host)
```

Comment:
Let us postpone this change to later, if it is not proven to be causing a failure.